packages/website/components/Topbar/Topbar.tsx

```diff
@@ -99,6 +105,14 @@ export function Topbar({ links }: TopbarProps) {
 
         <Flex className={styles.docSearchContainer}>
           <DocSearch />
+          {session ? (
+            <Button
+              className={styles.signOut}
+              onClick={() => signOut({ callbackUrl: '/' })}
+            >
+              Log out
+            </Button>
+          ) : null}
```

Comment:
Do we want a log in button?

Reply:
No, on the last Forma meeting with designers we agreed that we won't add any links/buttons for login. So it's just URL that you have to remember basically.